checks.d/docker_daemon.py

```diff
@@ -357,6 +358,19 @@ def _get_tags(self, entity=None, tag_type=None):
         if entity is not None:
             pod_name = None
 
+            if self.include_app_id and (tag_type is CONTAINER or tag_type is PERFORMANCE):
+                self.log.debug("We need to capture Marathon Id")
+                cont_id = entity.get("Id")
+                if cont_id is not None:
+                    cont_inspected = self.docker_client.inspect_container(cont_id)
+                    for env_var in cont_inspected['Config']['Env']:
+                        self.log.debug("We have env_var : %s" % env_var)
+                        if 'MARATHON_APP_ID=' in env_var:
+                            app_id = env_var.replace('MARATHON_APP_ID=','')
+                            self.log.debug("We have found the id as %s" % app_id)
+                            tags.append("app_id:%s" % app_id)
```

Comment:
Same as for the attribute, `app_id` is quite generic, it will be difficult to make a sense of it in the check as we support many orchestrators besides Marathon. Could you rename it to `marathon_app_id` please?